x/market/internal/types/pool_info.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// PoolInfo defines information to describe each denomination pool
```

Comment:
Comment is not informative; "describe each denomination pool" to describe an "info pool" is not sufficiently descriptive

Reply:
Removed